lib/gems/pending/appliance_console/cli.rb

```diff
@@ -117,6 +122,7 @@ def parse(args)
         opt :username, "Database Username",  :type => :string,  :short => 'U', :default => "root"
         opt :password, "Database Password",  :type => :string,  :short => "p"
         opt :dbname,   "Database Name",      :type => :string,  :short => "d", :default => "vmdb_production"
+        opt :db_hourly_maintenance, "Configure database hourly maintenance", :type => :bool, :short => :none
```

Comment:
You can just not specify the `:short` key, we do it with things like `:logdisk` and `:dbdisk` if you need an example.